contracts/Pool.sol

```diff
@@ -243,7 +243,7 @@ contract Pool is PoolFDT {
     */
     function _handleDefault(address loan, uint256 defaultSuffered) internal {
 
-        (uint256 bptsBurned, uint256 postBurnBptBal, uint256 liquidityAssetRecoveredFromBurn) = PoolLib.handleDefault(liquidityAsset, stakeLocker, stakeAsset, loan, defaultSuffered);
```

Comment:
Lets actually take this out and add it into the PR for this: https://app.clubhouse.io/maplefinance/story/2038/maple-core-c4-gas-optimizations